app/assets/stylesheets/custom.css.scss

```diff
@@ -302,3 +300,68 @@ table td {
     margin-right: 1em;
   }
 }
+
+.index-container img {
+  text-align: center;
+  padding-top: 4px;
+}
+
+.index-container h4 {
+  min-height: 50px;
+}
+
+.index-container h2 {
+  text-align: center;
+}
+
+.description {
+  text-align: center;
+  font-size: 20px;
+}
+
+.thumbnail {
+  width: 300px;
+  margin-left: 33%;
+  opacity: 0.7;
+  background-color: cornsilk;
```

Comment:
Color `cornsilk` should be written in hexadecimal form as `#fff8dc`<br>Color literals like `cornsilk` should only be used in variable declarations; they should be referred to via variable everywhere else.